Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.elastic.co/kibana/kibana:6.5.1
+FROM docker.elastic.co/kibana/kibana:6.7.1
 
 RUN export NODE_OPTIONS="--max-old-space-size=1536" && \
-    bin/kibana-plugin install https://github.com/pjhampton/kibana-prometheus-exporter/releases/download/6.5.1/kibana-prometheus-exporter-6.5.1.zip
+    bin/kibana-plugin install https://github.com/pjhampton/kibana-prometheus-exporter/releases/download/6.7.1/kibana-prometheus-exporter-6.7.1.zip
```

Comment:
Worthwhile making the version bits into PARAMs?
DRY this a tiny bit?

Reply:
I don't think this is worth it as it can easily be a different version depending on the exporter being updated. Currently, it matches, but if there is a bug this will not always match the Elasticsearch version. I get the link to use from https://github.com/vvanholl/elasticsearch-prometheus-exporter#Install (Slight variation because I prefer to be exact).

Reply:
Updating as per their documentation.

Reply:
Read https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact to understand why I can't use a variable for `FROM` (scoping issues).

Reply:
Yeah, I knew about the FROM bit ;)